BUILD

```diff
@@ -24,12 +24,26 @@ As openMVG use some C++11 features you must have a c++11 ready compiler:
 - GCC >= 4.7
 
 --------------------------
-General informations
-for openMVG cmake options
+CMake Options
 --------------------------
-OpenMVG_BUILD_TESTS (ON/OFF(default))=> Build openMVG unit tests
-OpenMVG_BUILD_EXAMPLES (ON/OFF(default))=> Build OpenMVG example applications.
-  Does not affect binaries under 'software'
+
+OpenMVG_USE_BOOST (default ON): Use Boost library (enable modules like localization/voctree and other features and optimizations.
+BOOST_NO_CXX11 (default OFF): If your Boost binaries are compiled without C++11 support, you need to set this option to avoid compilation errors.
+                              This is most likely to be the case if you use the system packages to install boost.
+OpenMVG_USE_OPENMP (default ON): Use OpenMP parallelization (huge impact on performances)
+OpenMVG_USE_CCTAG (default ON): Build with CCTag markers support.
+OpenMVG_USE_OPENGV (default OFF): Build with openGV for multi-cameras localization.
+OpenMVG_USE_ALEMBIC (default OFF): Build with Alembic file format support
```

Comment:
this is tougher as it requires several definition
```
ALEMBIC_ROOT
ALEMBIC_HDF5_ROOT
ALEMBIC_ILMBASE_ROOT
ALEMBIC_OPENEXR_ROOT
```

On the other hand, if we switch to the latest version of alembic we can get rid of our FindAlembic and use the AlembicConfig.cmake that is generated with all the needed dependencies. In this case we just need to define `-DAlembic_DIR=/alembici/install/prefix/lib/cmake/Alembic/`  (where AlembicConfig.cmake can be found)